extensions/sql-database-projects/src/test/projectTree.test.ts

```diff
@@ -68,24 +68,24 @@ describe('Project Tree tests', function (): void {
 
 		const tree = new ProjectRootTreeItem(proj);
 		should(tree.children.map(x => x.uri.path)).deepEqual([
-			'/TestProj/Database References',
-			'/TestProj/duplicateFolder',
-			'/TestProj/someFolder',
-			'/TestProj/duplicate.sql']);
+			'/TestProj.sqlproj/Database References',
```

Comment:
This is not affecting the display name for projects right?

Reply:
nope, I added a line to set the label of the project node to the project name without the .sqlproj extension
https://github.com/microsoft/azuredatastudio/blob/67b2505b48681b8314e65ebbc817a8c5ec2914c8/extensions/sql-database-projects/src/models/tree/projectTreeItem.ts#L51